pyco_proc.py

```diff
@@ -23,6 +52,11 @@ def proc_res(fd, args):
     engines = list()
     engines_outs = dict()
     results = dict()
+
+    current_time = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
+    if args.stats and args.stats == StatsDestination.SEPARATE_FILES:
+        Path(f"./stats/{current_time}/").mkdir(parents=True, exist_ok=True)
```

Comment:
What is this for?

Reply:
When an appropriate flag is set (by default set to the CSV file on stdout), you can export the statistics to JSON files at this location instead of printing them to the CSV file (to the stdout).